src/components/proofs/ERC721ProofSection.tsx

```diff
@@ -56,18 +56,18 @@ export default function ERC721ProofSectionSuspended({
   network: Network
 }) {
   return (
-    <Suspense
-      fallback={
-        <ProofSection title={<BodyText>{network} NFTs</BodyText>}>
+    <ProofSection title={<BodyText>{network}</BodyText>}>
```

Comment:
Simplified this part a little, we repeated the `ProofSection` thing twice, now only once